hZookeeper

beenum22 opened the pull request:
All the files except the hydra libraries which needs to be added as submodule. Need to make few changes to case_2 in it. Case_1 and Case_3 works fine.

Commit:
Add files via upload

Commit:
Create extra.md

Commit:
Create extra.md

Commit:
images

Commit:
test cases

Commit:
Delete extra.md

Commit:
Delete extra.md

Commit:
Update README.md

kratos7 commented on src/case_1/zk_client.py:
```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
```
Please follow the author convention as in other hydra projects.

beenum22 replied:
@kratos7  Okay.

kratos7 commented on src/case_1/zk_client.py:
```diff
@@ -0,0 +1,192 @@
+
+class ZKPub(HDaemonRepSrv):
+	def __init__(self,port,run_data,zk_server_ip):
+		self.run_data = run_data
```
Use 4 space indent everywhere.

kratos7 replied:
Also please add comments to functions for better understanding of the code.

affansyed replied:
I have asked him to run pip8 and fix all syntax error. Should be done shortly.

beenum22 replied:
@kratos7 Okay.

kratos7 commented on README.md:
```diff
@@ -1,2 +1,256 @@
+
+##Background:
+
+###Mesos/Marathon: 
```
All of this is some great documentation. I think it is general enough to be put into a separate docs repo/folder under lake-lerna ? or may be move to hHelloWorld.
@sushilks   what do you think ?

sushilks replied:
yes good idea, it should be moved to hello world.

Commit:
Minor changes

added 4 space indentation, added comments, made minor changes to the
code of case_3, case_2 is still under development

beenum22 commented:
@kratos7 Added the 4 space indentation and author convention as I was advised.

Commit:
Update README.md
